app/test_setup.go

```diff
@@ -36,8 +45,15 @@ func InitElysTestApp(initChain bool) *ElysApp {
 		MakeEncodingConfig(),
 		simapp.EmptyAppOptions{},
 	)
+
+	return app
+}
+
+// Initializes a new ElysApp without IBC functionality
```

Comment:
This comment is same as one on top of InitiateNewElysApp

Reply:
yeah, I'll take a look at it and put proper comments.